python/tank/context.py

```diff
@@ -1051,7 +1051,7 @@ def from_entity(tk, entity_type, entity_id):
         task_context = _task_from_sg(tk, entity_id)
         context.update(task_context)
 
-    elif entity_type in ["PublishedFile", "TankPublishedFile"]:
+    elif entity_type in ["PublishedFile", "TankPublishedFile", "Version"]:
```

Comment:
Will the `find_one` work for `Version` since the task field is `sg_task` instead of `task` (line 1058)?